mungegithub/mungers/matchers/comment/command_test.go

```diff
@@ -45,6 +45,18 @@ func TestParseCommand(t *testing.T) {
 			comment:         "/COMMAND Valid command",
 		},
 		{
+			expectedCommand: &Command{Name: "COMMAND", Arguments: "Multiple Lines"},
```

Comment:
I'd like commands to be at the beginning of comments.  Easier for humans to parse and figure out where bot commands came from.